buildscripts/kokoro/android-interop.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -exu -o pipefail
+cat /VERSION
```

Comment:
https://github.com/ericgribkoff/grpc-java/blob/9b9c6fc5fbd998a6e4073902d5524a4a634491c0/buildscripts/kokoro/release_artifacts.sh#L4

Something like this will make the script easier to run on our workstations, because /VERSION only exists in kokoro and the flags on line 3 make the script exit on any command failure.

Reply:
Done